### PR TITLE
Update logging images to match 3.7.0 upstream

### PR DESCRIPTION
**Pull request by nickgerace**
Update logging images to match Banzai Logging Operator 3.7.0 helm chart
images. This includes four images: logging-operator, fluentd, fluentbit,
and fluentbit-debug.

#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexographically) 
- [x] Changes to scripting or CI config have been tested to the best of your ability

#### Types of Change ####

version bump

#### Linked Issues ####

Issue: [#29642](https://github.com/rancher/rancher/issues/29642)

#### Additional Notes ####

n/a

### CODE REVIEW

**Review by paynejacob — CHANGES_REQUESTED** [review]
We should not remove existing image tags.

**Review by paynejacob — APPROVED** [review]

**Review by mrajashree — DISMISSED** [review]

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

**Review by cbron — DISMISSED** [review]

**Comment by nickgerace** [comment]
Small cosmetic change. Will merge after first new approval.

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**
